src/Microsoft.TemplateEngine.Abstractions/Microsoft.TemplateEngine.Abstractions.csproj

```diff
@@ -4,5 +4,6 @@
         <TargetFrameworks Condition="'$(PackSpecific)' == 'true'">$(NETStandardTargetFramework)</TargetFrameworks>
         <Description>Contracts for extending Template Engine</Description>
         <IsPackable>true</IsPackable>
+        <EnableAnalyzers>true</EnableAnalyzers>
```

Comment:
I realize this PR has been merged, but this change is impacting source-build.  The changes like this which explicitly set EnableAnalyers, overrides the value specified in eng/Analyzers.props.  This nuance impacts source-build which disables the analyers because they aren't part of source-build.